src/Controller/test/UsersControllerTest.ts

```diff
@@ -2,30 +2,43 @@ import { SettingRepositoryInterface } from '../../Domain/Setting/SettingReposito
 import { SettingRepostioryStub } from '../../Domain/Setting/test/SettingRepositoryStub'
 import { GetSettingTest } from '../../Domain/UseCase/GetSetting/test/GetSettingTest'
 import { GetSettingsTest } from '../../Domain/UseCase/GetSettings/test/GetSettingsTest'
+import { GetUserKeyParamsTest } from '../../Domain/UseCase/GetUserKeyParams/test/GetUserKeyParamsTest'
 import { UpdateUser } from '../../Domain/UseCase/UpdateUser'
+import { KeyParamsFactoryInterface } from '../../Domain/User/KeyParamsFactoryInterface'
+import { KeyParamsFactoryStub } from '../../Domain/User/test/KeyParamsFactoryStub'
+import { UserRepostioryStub } from '../../Domain/User/test/UserRepostioryStub'
+import { UserRepositoryInterface } from '../../Domain/User/UserRepositoryInterface'
 import { SettingProjector } from '../../Projection/SettingProjector'
 import { SettingProjectorTest } from '../../Projection/test/SettingProjectorTest'
 import { UsersController } from '../UsersController'
 
 export class UsersControllerTest {
   static makeSubject({
     updateUser,
-    repository = new SettingRepostioryStub([]),
+    settingRepository = new SettingRepostioryStub([]),
     projector = SettingProjectorTest.get(),
+    userRepository = new UserRepostioryStub([]),
+    keyParamsFactory = new KeyParamsFactoryStub({ version: '004', identifier: 'test@test.com' })
   }: {
     updateUser: UpdateUser,
-    repository?: SettingRepositoryInterface,
+    settingRepository?: SettingRepositoryInterface,
     projector?: SettingProjector,
+    userRepository?: UserRepositoryInterface,
+    keyParamsFactory?: KeyParamsFactoryInterface
```

Comment:
Just a hint and to justify my habit: trailing commas for anything multiline are really practical -- easy to add new lines at the end, move lines around and last but not least -- better diffs. E.g. here the line with `projector` is rightfully not highlighted -- because it didn't change. If there were no trailing comma though, it (unnecessarily) would be.

A related linter rule: https://eslint.org/docs/rules/comma-dangle

Lots of fun to be had with commas in grammars in general.
Same with semicolons. And some other similar tokens.

// This is why they don't exist in TAO. And it's pretty sweet.

Reply:
Will add to @standardnotes/config and update this PR

Reply:
Hmm there's a lot of linter errors after the upgrade - will apply in a separate PR so it doesn't obfuscate this one